### PR TITLE
fix UI issues INVGEN-47281, INVGEN-47280

### PR DESCRIPTION
**Pull request by Houmicek**
- Added BOM icon
- Cloud icon has default color

### CODE REVIEW

**Review by obergrd — APPROVED** [review]

File: WebApplication/ClientApp/src/components/downloads.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ export class Downloads extends Component {
                 // register the row
                 data.push({
                     id: 'bom',
-                    icon: 'products-and-services-24.svg',
+                    icon: 'file-spreadsheet-24.svg',
```
Comment (erymski) [inline WebApplication/ClientApp/src/components/downloads.js:132]: Maybe you can use `FileSpreadsheet24` icon from @hig/icons package? (https://github.com/Autodesk/hig/blob/development/packages/icons/src/icons/information-dense/file-spreadsheet.svg)

File: WebApplication/ClientApp/src/components/parametersContainer.css [hunk 2]
```diff
@@ -108,7 +108,8 @@ div.changedOnUpdate-input-wrapper {
     white-space: pre-wrap;
 }
 
-/* trick to change SVG color. https://stackoverflow.com/a/49627345 */
+/* trick to change SVG color. https://stackoverflow.com/a/49627345
```
Comment (erymski) [inline WebApplication/ClientApp/src/components/parametersContainer.css:111]: Could you please validate that the icon for Update button for cases with adoption warnings is not changed?
This change was done on purpose in #361

Comment (obergrd) [reply WebApplication/ClientApp/src/components/parametersContainer.css:111]: right, verify it please

Comment (Houmicek) [reply WebApplication/ClientApp/src/components/parametersContainer.css:111]: I have checked it , warning has correct color

File: WebApplication/ClientApp/src/components/downloads.js (hunk 1 above)
Comment (Houmicek) [reply WebApplication/ClientApp/src/components/downloads.js:132]: this icon is recommended by Martin

**Pull request merged**